src/dto/facebookLoginDTO.ts

```diff
@@ -0,0 +1,24 @@
+import { Gender } from '@constants/users/attributes.constants';
+import { IsEmail, IsEnum, IsNotEmpty, IsString } from 'class-validator';
+
+export class FacebookLoginDTO {
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsString()
+  @IsNotEmpty()
+  firstName: string;
+
+  @IsString()
+  @IsNotEmpty()
+  lastName: string;
+
+  @IsString()
+  @IsNotEmpty()
+  facebookID: string
```

Comment:
Here you could take care of the max length of these strings, just to avoid a _character limit exceeded_  from the DB if someone sends a super large string. What do you think?